### PR TITLE
ASP.Net Core 2.1 Formatter

### PR DESCRIPTION
**Pull request by Tornhoof**
- new nuget package
- tests

### CODE REVIEW

File: SpanJson.AspNetCore.Formatter/SpanJsonOutputFormatter.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            SupportedMediaTypes.Add("application/json");
+            SupportedMediaTypes.Add("text/json");
+            SupportedMediaTypes.Add("application/*+json");
+        }
+
+        public override async Task WriteResponseBodyAsync(OutputFormatterWriteContext context)
+        {
+            if (context.Object != null)
+            {
+                await JsonSerializer.NonGeneric.Utf8.SerializeAsync<TResolver>(context.Object, context.HttpContext.Response.Body).ConfigureAwait(false);
```
Comment (khellang) [inline SpanJson.AspNetCore.Formatter/SpanJsonOutputFormatter.cs]: This could avoid the async state machine by just returning the task, no?

Comment (Tornhoof) [reply SpanJson.AspNetCore.Formatter/SpanJsonOutputFormatter.cs]: SerializeAsync is returning a ValueTask, so a simple return JsonSerializer... does not work.

Comment (khellang) [reply SpanJson.AspNetCore.Formatter/SpanJsonOutputFormatter.cs]: `.AsTask()`?

Comment (Tornhoof) [reply SpanJson.AspNetCore.Formatter/SpanJsonOutputFormatter.cs]: Good point, I have to try that out.

**Pull request merged**